scripts/start_avd.py

```diff
@@ -0,0 +1,102 @@
+import sys
+import os
+import platform
+import subprocess
+import django
+from os.path import dirname, abspath
+
+MobSF_path = dirname(dirname(abspath(__file__)))
```

Comment:
Is this already available from `settings.py`?

Reply:
What do you mean?